app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+<div class='sign-in'>
+  <div class="field">
+    <%= f.text_field :name, placeholder: 'Name' %>
+  </div>
+
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: 'Email' %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", placeholder: 'Password' %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
```

Comment:
**[OPTIONAL]**


- Your devise works well, however, You can improve the login page by adding a link to the sign up so that users that have not signed up can click the link so it takes them to the sign up page. You can add this after line 22
```
  <br>
  <div class="">
    <%= link_to "Sign up", new_user_registration_path %>
  </div>
```


*Your sign-in page could be improved so there's a sign-up link*
![Screenshot from 2023-03-09 16-03-10](https://user-images.githubusercontent.com/59397159/224069548-7fe3205f-e7b2-4cdb-993b-c6ead3db81b1.png)

*The added sign up link, this improve the user experience*
![Screenshot from 2023-03-09 16-21-11](https://user-images.githubusercontent.com/59397159/224070015-064ce969-b267-410c-94b1-de347a62ba59.png)